### PR TITLE
Remove references to cppw, cpgr

### PR DESCRIPTION
**Pull request by zeha**
cppw, cpgr were Debian-only tools, which I've dropped in a recent upload.

Upstream should have never had tests for them.

Signed-off-by: Chris Hofstaedtler <zeha@debian.org>

### CODE REVIEW

**Review by ikerexxe — APPROVED** [review]
Wow, that's some serious weight lightening.

**Comment by alejandro-colomar** [comment]
+1

**Comment by alejandro-colomar** [comment]
I think there's some stuff that should also be removed:

```sh
alx@debian:~/src/shadow/shadow/cp$ grep -rn cptools
tests/run_all.coverage:853:run_test ./cptools/01/run1
tests/run_all.coverage:854:run_test ./cptools/01/run2
tests/run_all.coverage:855:run_test ./cptools/01/run3
tests/run_all.coverage:856:run_test ./cptools/01/run4
tests/run_all:837:run_test ./cptools/01/run1
tests/run_all:838:run_test ./cptools/01/run2
tests/run_all:839:run_test ./cptools/01/run3
tests/run_all:840:run_test ./cptools/01/run4
alx@debian:~/src/shadow/shadow/cp$ find | grep cptools
alx@debian:~/src/shadow/shadow/cp$ 
```

Those lines are referencing tests that don't exist anymore.

**Comment by zeha** [comment]
> I think there's some stuff that should also be removed:

Indeed. Removed now.

File: tests/run_all [hunk 1]
```diff
@@ -845,12 +834,6 @@ run_test ./usertools/usermod/49_usermod_change_gid+move_homedir_other_device/use
 run_test ./usertools/usermod/50_usermod_change_uid+move_homedir/usermod.test
 run_test ./usertools/usermod/51_usermod_change_gid+move_homedir/usermod.test
 run_test ./usertools/usermod/52_usermod_move_homedir_symlink/usermod.test
-run_test ./cptools/01/run1
-run_test ./cptools/01/run2
-run_test ./cptools/01/run3
-run_test ./cptools/01/run4
-run_test ./cktools/01/run1
-run_test ./cktools/01/run2
```
Comment (alejandro-colomar) [inline tests/run_all:853]: cktools tests still exist, no?

Comment (zeha) [reply tests/run_all]: Sigh, yes. Shouldn't do this in the middle of 5 other things.

Comment (alejandro-colomar) [reply tests/run_all]: :)

**Pull request merged**